rm -f glxinfo
gcc -m32 -o glxinfo -O2 -fno-strength-reduce -fno-strict-aliasing -std=gnu9x -pedantic
    -Wall -Wpointer-arith -Wundef     -L/home/users/pluto/rpm/BUILD/X11-6.8.2/xc/exports/lib
    glxinfo.o -lGLU -lGL -lXext -lX11  -lpthread -lm
    -Wl,-rpath-link,/home/users/pluto/rpm/BUILD/X11-6.8.2/xc/exports/lib
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_DeleteException@GCC_3.0'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_Resume@GCC_3.0'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_RaiseException@GCC_3.0'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_GetRegionStart@GCC_3.0'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_GetDataRelBase@GCC_3.0'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_SetGR@GCC_3.0'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_GetLanguageSpecificData@GCC_3.0'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_Resume_or_Rethrow@GCC_3.3'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_GetIP@GCC_3.0'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_GetTextRelBase@GCC_3.0'
/usr/bin/../lib/libstdc++.so.6: undefined reference to `_Unwind_SetIP@GCC_3.0'
collect2: ld returned 1 exit status
make: *** [glxinfo] B³±d 1

--- X11-6.8.2/xc/programs/glxinfo/Imakefile.orig	2004-04-23 21:54:33.000000000 +0200
+++ X11-6.8.2/xc/programs/glxinfo/Imakefile	2005-03-31 20:08:04.000000000 +0200
@@ -5,7 +5,7 @@
 #if BuildGLULibrary
 
         DEPLIBS = $(DEPGLULIB) $(DEPGLXLIB) $(DEPXLIB)
-LOCAL_LIBRARIES = $(GLULIB) $(GLXLIB) $(XLIB) 
+LOCAL_LIBRARIES = $(GLULIB) -lstdc++ $(GLXLIB) $(XLIB)
         DEFINES = -DDO_GLU
 
 #else
